ramls/inventory-batch.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title: Inventory API
-version: v0.3
+version: v0.4
```

Comment:
@marcjohnson-kint  why to change interface version if nothing added in API declaration ?
We change minor number if things only added to the interface – e.g. a new resource or method on existing resources